agent/security_helper.py

```diff
@@ -148,14 +149,17 @@ def selinux_status():
     '/etc/shadow',
     '/etc/group'
 ]
+
 SSHD_CONFIG_PATH = '/etc/ssh/sshd_config'
-SSHD_CONFIG_DEFAULTS = {
-    'PermitEmptyPasswords': 'no',
-    'PermitRootLogin': 'yes',
-    'PasswordAuthentication': 'yes',
-    'AllowAgentForwarding': 'yes',
-    'Protocol': '2,1'
+# Value: (default, safe).
+SSHD_CONFIG_PARAMS_INFO = {
+    'PermitEmptyPasswords': ['no', 'no'],
+    'PermitRootLogin': ['yes', 'no'],
+    'PasswordAuthentication': ['yes', 'no'],
```

Comment:
In some newer versions the default for PasswordAuthentication is "no". See for example
https://www.freebsd.org/cgi/man.cgi?query=sshd_config&apropos=0&sektion=5&manpath=FreeBSD+12.0-RELEASE&arch=default&format=html
Also in older versions the default for Protocol is "2,1".